components/embedder_support/user_agent_utils_unittest.cc

```diff
@@ -56,4 +56,13 @@ TEST(UserAgentUtilsTest, UserAgentFromCommandLine) {
   EXPECT_EQ(brave_metadata, empty_metadata);
 }
 
+#if BUILDFLAG(IS_LINUX)
+TEST(UserAgentUtilsTest, UserAgentLinuxKernelVersion) {
+  auto metadata = GetUserAgentMetadata();
+
+  EXPECT_TRUE(metadata.platform == "Linux");
+  EXPECT_TRUE(metadata.platform_version == "");
```

Comment:
please use `EXPECT_EQ(...)` so we can see the values in the test output if it fails.

Reply:
Done